### PR TITLE
[TECHNICAL-SUPPORT] LPS-69722

### PR DESCRIPTION
**Pull request by jonathanmccann**
/cc @Alec-Shay

### CODE REVIEW

File: modules/apps/collaboration/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/edit_page.jsp [hunk 1]
```diff
@@ -217,7 +217,12 @@ if (portletTitleBasedNavigation) {
 
 							<%
 							try {
-								wikiEngineRenderer.renderEditPageHTML(selectedFormat, pageContext, node, wikiPage);
+								if (wikiPage.isNew() && (templatePage != null)) {
```
Comment (sergiogonzalez) [inline modules/apps/collaboration/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/edit_page.jsp]: Hey @jonathanmccann @Alec-Shay I think that wikiPage could be null right? What should we do in that case? Can you please resend considering that scenario?
Thanks!

**Comment by sergiogonzalez** [comment]
Can you please resend? Thx!

**Comment by Alec-Shay** [comment]
Hi @sergiogonzalez,

I've tested with this, but I could not find a use case where wikiPage is null. Theoretically it would be possible if it failed this check in `edit_page.jsp`: 

```
...
else if ((wikiPage == null) && editTitle) {
	editable = true;
	wikiPage = new WikiPageImpl();
	wikiPage.setNew(true);
	wikiPage.setNodeId(node.getNodeId());
	wikiPage.setFormat(selectedFormat);
	wikiPage.setParentTitle(parentTitle);
}
```

... but I could not find a use case where both of these conditions were satisfied.

In any case, `renderEditPageHTML()` does not seem to cause any problems if null is passed into it, so I think it would be fine if it were still handled this way. A simple null check could be added to prevent NPE's with `wikiPage.isNew()`, but apart from that I don't see much of another way that this could be handled better.

I will resend with that extra check, but it's a very small change. Let me know if you disagree with it though. Thank you.

**Comment by Alec-Shay** [comment]
Re-sent to Sam: https://github.com/SamZiemer/liferay-portal/pull/148

**Pull request closed without merging**